packages/cxl-lumo-styles/package.json

```diff
@@ -13,6 +13,7 @@
     "@vaadin/icon": "^23.3.7",
     "@vaadin/polymer-legacy-adapter": "^23.3.7",
     "@vaadin/vaadin-lumo-styles": "^23.3.7",
-    "@vaadin/vaadin-themable-mixin": "^23.3.7"
+    "@vaadin/vaadin-themable-mixin": "^23.3.7",
+    "sass-embedded": "^1.77.8"
```

Comment:
Hm, so we can't re-use things available at https://github.com/conversionxl/aybolit/blob/master/packages/sass-render/package.json here?

Reply:
There's not much to reuse, it would be mostly copy/paste. `sass-embedded` provides a performance benefit. We don't need to interpolate templates, or add features which are already baked in. The `sass-embedded` CLI is a pretty straight forward way to get things done

Reply:
I think we should not compile it in `aybolit` but just provide a SASS file which is later compiled in themes: https://github.com/conversionxl/cxl-wpstarter/blob/0a44eaf3e29137218d2959a0d672d39ad9ff6b6d/packages/institute-theme/package.json#L23

Thoughts @lkraav ?